hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/mod/SubmitModificationsOp.java

```diff
@@ -67,4 +75,12 @@ protected boolean submitOp(@NonNull final HapiSpec spec) throws Throwable {
                         .toArray(HapiSpecOperation[]::new))));
         return false;
     }
+
+    private HapiTxnOp<?> originalTransaction() {
```

Comment:
What does this method name mean when at L81-83 you actually change the payer?  Not "original" anymore?

Reply:
Added javadoc.